src/libraries/System.Net.Quic/src/System.Net.Quic.csproj

```diff
@@ -11,6 +11,7 @@
     <DefineConstants Condition="'$(TargetPlatformIdentifier)' == 'windows'">$(DefineConstants);TARGET_WINDOWS</DefineConstants>
     <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetPlatformIdentifier)' == ''">SR.SystemNetQuic_PlatformNotSupported</GeneratePlatformNotSupportedAssemblyMessage>
     <ApiExclusionListPath Condition="'$(TargetPlatformIdentifier)' == ''">ExcludeApiList.PNSE.txt</ApiExclusionListPath>
+    <UseQuicTransportPackage  Condition="'$(UseQuicTransportPackage)' == ''">false</UseQuicTransportPackage>
```

Comment:
Is this ever set to `true`? It looks like a newly added property which is always set to false.

Reply:
yes, at the moment. You could pass `/p:UseQuicTransportPackage=true` from command line to test changes without messing up with the project file. I expect this will change in `main` eventually (so we can consume pre-releases from msquic) and it would stay false in `release/7.0`